packages/next/src/build/index.ts

```diff
@@ -293,6 +293,7 @@ export default async function build(
 
       const { headers, rewrites, redirects } = customRoutes
       NextBuildContext.rewrites = rewrites
+      NextBuildContext.redirects = config._originalRedirects
```

Comment:
Might want to also rename this to `NextBuildContext.originalRedirects` and similar for `rewrites` as they also get transformed.

Reply:
`rewrites` was already there for a while, fine to rename it?

Reply:
We should probably keep the original field and add a separate one